ext/http/01_http.js

```diff
@@ -471,4 +468,76 @@ function upgradeHttp(req) {
   return req[_deferred].promise;
 }
 
+const spaceCharCode = StringPrototypeCharCodeAt(" ", 0);
```

Comment:
I feel like it's reasonable to only check for plain old spaces instead of all whitespace? I couldn't find anywhere that explained exactly how this is supposed to work though. See pass git history when it was checking all kinds of whitespace (and still faster).

Reply:
https://greenbytes.de/tech/webdav/rfc7230.html#whitespace

At least that particular spec website defines that horizontal tab is also allowed as whitespace.